sba-business-rules.md

```diff
@@ -0,0 +1,37 @@
+---
+title: DATA Act business rules
+---
+As an SBA user, I need to see the business rules being applied to my source data
+
+Business rules or validation rules will be applied to ensure that submitted data by the US Small Business Administration meets the DATA Act criteria. Goals of these validation rules will improve cleanliness, accuracy, and meaningfulness of the data set. Below is a draft of proposed rules and are subject to change. 
```

Comment:
It might be helpful to be specific about when the rules will be applied. For example, "agency submissions will be checked against these rules before the data submission is accepted by Treasury" or something like that.